modules/ROOT/pages/upgrade-policies-proxies.adoc

```diff
@@ -0,0 +1,128 @@
+= Upgrade Java for Policies and API Proxies 
+
+Upgrade your policies and API proxies to be compatible with Java 17.
+
+[[upgrading-automated-policies]]
+== Upgrading Automated Policies
+
+Upgrade your automated policies (included and custom) to Java 17.
+
+[[upgrading-included-automated-policies]]
+=== Upgrading Included Automated Policies
+
+To upgrade xref:gateway::policies/policies-availability-by-gateway.adoc[MuleSoft included policies] that are automated to Java 17:
+
+. In API Manager, go to *Automated Policies*.
+. Upgrade your included policies:
+ .. In *Policy version*, select the latest version of the included policy.
+ .. In *Rule of application*:
+  * To apply included policies to all Mule runtimes, select *All runtimes* and click *Apply*.
+  * To apply the included policies to Flex Gateways only, select *Flex Gateways only* and click *Apply*.
+  * To apply the included policies to Mule Gateways only:
+  ** Select *Mule Gateways only*.
+  ** In *From version* and *To*, enter the Mule version range and in *Java versions*, select *All Java versions* or *Choose specific Java versions*. 
+  ** Click *Apply*.
+
+[[upgrading-custom-automated-policies]]
+=== Upgrading Custom Automated Policies
+
+To upgrade custom automated policies to Java 17:
+
+. In API Manager, go to *Automated Policies*.
+. Upgrade the required extensions and connectors to Java 17.
+ * If your custom policy has Anypoint Connectors, refer to xref:java-support.adoc#anypoint-connectors-and-modules[Upgrading Anypoint Connectors and Modules].
+ * If your custom policy has custom connectors, refer to <<Upgrading Custom Connectors INSERT LINK>>.
+. xref:gateway::policies-custom-upgrade.adoc[Upgrade the custom policy (link to new page in Gateway)].
+. xref:gateway::policies-custom-package.adoc[Package the custom policy].
+. xref:gateway::policies-custom-upload-to-exchange.adoc[Republish the custom policy to Exchange].
+. Apply the new version of the custom policy that is compatible with Java 17 to the API instance:
+ .. Go to *Policies*.
+ .. In *Applied policy status*, verify that the latest definition is applied.
+ * If the latest definition is applied, click on the three dots menu and select *Check for implementation updates*.
```

Comment:
The three dots menu is referred to as More <noun> (...) menu

So like "more options (...) menu or whatever.